database/migration/aruha-400-schema-evolution.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS zn_data.event_type_schema (
+  ets_event_type_name varchar(255) NOT NULL REFERENCES zn_data.event_type (et_name),
+  ets_schema_object jsonb NOT NULL
+);
+
+CREATE INDEX ON zn_data.event_type_schema USING gin (ets_schema_object);
+
+CREATE INDEX ON zn_data.event_type_schema ((ets_schema_object->>'version'));
+CREATE INDEX ON zn_data.event_type_schema ((ets_schema_object->>'created_at'));
+
+CREATE UNIQUE INDEX ON zn_data.event_type_schema ((ets_schema_object->>'version'),
+                                                  (ets_event_type_name));
+
+UPDATE zn_data.event_type
+SET et_event_type_object  = jsonb_set(et_event_type_object, '{schema,version}', '"1.0.0"', false);
```

Comment:
Hun, that's a problem. I just checked AWS documentation and they claim to have 9.5. Maybe we are using an outdated Postgres? I prefer to upgrade it than change the code =)